EventsExpress.Core/Services/PhotoService.cs

```diff
@@ -51,6 +53,55 @@ public async Task<Photo> AddPhoto(IFormFile uploadedFile)
             return photo;
         }
 
+        public async void AddAvatar(Guid userId, string url)
+        {
+            var user = _db.UserRepository.Get("Photo").FirstOrDefault(u => u.Id == userId);
```

Comment:
What happens if user is null? If you have a userId and you need to ensure it exists - please use `Single` LINQ moethod.